From cd755e3f753cea6b0806d6e846e09b51883ceced Mon Sep 17 00:00:00 2001
From: Chaos <lgf42031@gmail.com>
Date: Thu, 11 Nov 2021 11:44:17 +0800
Subject: [PATCH 06/11] [PATCH] thread_gum_js_loop

---
 src/anti-anti-frida.py | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/src/anti-anti-frida.py b/src/anti-anti-frida.py
index 2615b976..472c184e 100644
--- a/src/anti-anti-frida.py
+++ b/src/anti-anti-frida.py
@@ -1,6 +1,7 @@
 import lief
 import sys
 import random
+import os
 
 if __name__ == "__main__":
     input_file = sys.argv[1]
@@ -19,4 +20,9 @@ if __name__ == "__main__":
         if "FRIDA" in symbol.name:
             symbol.name = symbol.name.replace("FRIDA", random_name)
 
-    binary.write(input_file)
\ No newline at end of file
+    binary.write(input_file)
+
+    # gum-js-loop thread
+    random_name = "".join(random.sample("abcdefghijklmn", 11))
+    print(f"[*] Patch `gum-js-loop` to `{random_name}`")
+    os.system(f"sed -b -i s/gum-js-loop/{random_name}/g {input_file}")
\ No newline at end of file
-- 
2.31.1

