From 4abafcbf1c73a868c930d5bf7012228c581716b3 Mon Sep 17 00:00:00 2001
From: Chaos <lgf42031@gmail.com>
Date: Thu, 11 Nov 2021 11:44:17 +0800
Subject: [PATCH 08/10] [PATCH] protocol_unexpected_command

---
 src/droidy/droidy-client.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/droidy/droidy-client.vala b/src/droidy/droidy-client.vala
index 1bc1c58b..ca45c26c 100644
--- a/src/droidy/droidy-client.vala
+++ b/src/droidy/droidy-client.vala
@@ -998,7 +998,7 @@ namespace Frida.Droidy {
 						case "OPEN":
 						case "CLSE":
 						case "WRTE":
-							throw new Error.PROTOCOL ("Unexpected command");
+							break; //throw new Error.PROTOCOL ("Unexpected command");
 
 						default:
 							var length = parse_length (command_or_length);
-- 
2.31.1

